.docker/docker-compose-ocr.yml

```diff
@@ -72,7 +72,7 @@ services:
 
   form-rec-custom-api:
     container_name: azure-cognitive-service-custom-api
-    image: mcr.microsoft.com/azure-cognitive-services/form-recognizer/custom-api
+    image: mcr.microsoft.com/azure-cognitive-services/form-recognizer/custom-api:2.1-preview
```

Comment:
Apparently :latest no longer exposes the Forms API, only the Layout API. We need the Forms API to be able to build and train a new VT2 model and to create a compose model.